rust/core-lib/src/whitespace.rs

```diff
@@ -85,11 +85,11 @@ fn gcd(a: usize, b: usize) -> usize {
     } else if b == 0 || a == b {
         a
     } else {
-        let (mut a, mut b, mut r) = (a, b, 0);
+        let (mut a, mut b, mut _r) = (a, b, 0);
```

Comment:
I'd just define `r` inside the loop since it doesn't carry over to the next iteration and is not used anywhere else. That way it doesn't even need to be mutable.